src/tests/ftest/recovery/cat_recov_core.yaml

```
@@ -0,0 +1,58 @@
+# change host names to your reserved nodes, the
+# required quantity is indicated by the placeholders
+hosts:
+  test_servers: 4
+# Note that subtests below can set their own timeout so this
+# should be a general average of all tests not including outliers
+# (I'm looking at you "rebuild tests")
```

We shouldn't copy these comments
```suggestion
hosts:
  test_servers: 4
```